Sources/CodegenCLI/Commands/Initialize.swift

```diff
@@ -11,11 +11,21 @@ public struct Initialize: ParsableCommand {
     abstract: "Initialize a new configuration with defaults."
   )
 
+  @Option(
+    name: [.long],
+    help: "DEPRECATED - Use --schema-namespace instead."
+  )
+  // When removing this property also do:
+  // - remove the initialization value for schemaNamespace
+  // - remove schemaName validation in validate()
+  // - remove mutating keyword from validate() signature
+  var schemaName: String?
+
   @Option(
     name: [.long, .customShort("n")],
     help: "Name used to scope the generated schema type files."
   )
-  var schemaName: String
+  var schemaNamespace: String = ""
```

Comment:
Required to let users continue to use the `init` command with the `--schema-name` option for now.